data/wanderer/wanderers middle.txt

```diff
@@ -3103,9 +3103,9 @@ mission "Wanderers: Sestor: Exiles 3"
 			`	Rek speaks with the Korath again, and then tells you, "They say that three of their 'world-ships' will travel with us to meet with the Wanderer leaders. If they are able to work out favorable terms, and are convinced that the Wanderers have peaceful intentions, they will go with us to the Sestor factory world and shut it down."`
 				accept
 	on accept
-		log "Succeeded in communicating with the Korath exiles, by landing on the one habitable planet in their territory. (It seems like the exiles might have a taboo against using weapons on that world.) The exiles agreed to send some ships to help disable the remaining Kor Sestor factory."
+		log "Succeeded in communicating with the Korath Exiles, by landing on the one habitable planet in their territory. (It seems like the Exiles might have a taboo against using weapons on that world.) The Exiles agreed to send some ships to help disable the remaining Kor Sestor factory."
 		fail "Wanderers: Sestor: Exiles 2"
-		event "wanderers: exiles peaceful"
+		event "wanderers: Exiles peaceful"
```

Comment:
```suggestion
		event "wanderers: exiles peaceful"
```
I don't think event names should get edited.

Reply:
Oh, you got this one already.

Reply:
Yeah that was me missing a single one after doing a mass text replacement